docs/kubernetes/how-to-deploy-jenkins-x-in-linode-kubernetes-engine/index.md

```diff
@@ -0,0 +1,365 @@
+  namespace: jx
+  provider: gke # <- you should change this line to "kubernetes"!
+```
+
+It's time to run the `jx boot` command again, but this time from within the `jenkins-x-boot-config` folder:
+
+```bash
+cd jenkins-x-boot-config
+jx boot
+```
+
+This time, the command stops at a different step:
+
+{{< output >}}
+Attempting to resolve version for boot config https://github.com/jenkins-x/jenkins-x-boot-config from https://github.com/jenkins-x/jenkins-x-versions.git
+Booting Jenkins X
+
+STEP: validate-git command: /bin/sh -c jx step git validate in dir: /tmp/jenkinsx/jenkins-x-boot-config/env
+
+STEP: verify-preinstall command: /bin/sh -c jx step verify preinstall --provider-values-dir="kubeProviders" in dir: /tmp/jenkinsx/jenkins-x-boot-config
+
+jx boot has only been validated on GKE and EKS, we'd love feedback and contributions for other Kubernetes providers
+? Continue execution anyway? (Y/n)
+{{</ output>}}
+
+_Yes_ is the default answer, and you should go ahead and press enter.
+
+The next question is the cluster name:
+
+{{< output >}}
+? Cluster name
+{{</ output>}}
+
+Enter `jenkins-x-lke`.
+
+{{< output >}}
+? Git Owner name for environment repositories
+{{</ output>}}
+
+You should use your GitHub username as the Git Owner.
+
+{{< output >}}
+Environment repos will be private, if you want to create public environment repos, please set environmentGitPublic to true in jx-requirements.yml
+? Comma-separated git provider usernames of approvers for development environment repository
+{{</ output>}}
+
+You can use your GitHub username as an approver and add any other team member in this question.
+
+Pay attention to the next question:
+
+{{< output >}}
+Locking version stream https://github.com/jenkins-x/jenkins-x-versions.git to release v1.0.529. Jenkins X will use this release rather than master to resolve all versions from now on.
+writing the following to the OWNERS file for the development environment repository:
+approvers:
+- <your GitHub username>
+reviewers:
+- <your GitHub username>
+WARNING: TLS is not enabled so your webhooks will be called using HTTP. This means your webhook secret will be sent to your cluster in the clear. See https://jenkins-x.io/docs/getting-started/setup/boot/#ingress for more information
+? Do you wish to continue? [? for help] (y/N)
+{{</ output>}}
+
+Notice how _No_ is the default answer.
+
+However, you should answer **Yes** and move on to the next question.
+
+In the next steps, `jx` configures several components in your cluster.
+
+It might take a while before you're requested to interact with the terminal.
+
+When you do, the next questions are about the Jenkins X admin:
+
+{{< output >}}
+? Jenkins X Admin Username (admin)
+? Jenkins X Admin Password
+{{</ output>}}
+
+You should choose a name and a password for the admin user.
+
+Next, you should create a Jenkins X Bot.
+
+The bot will help you triage issues and pull requests and execute chores.
+
+**You need a GitHub account for your bot.**
```

Comment:
This can go in the Before you Begin section

Reply:
Moved